test/integration/certificates/trigger_controller_test.go

```diff
@@ -390,30 +389,9 @@ func selfSignCertificateWithNotBeforeAfter(t *testing.T, pkData []byte, spec *cm
 // triggered on certificate events.
 func applyTestCondition(t *testing.T, ctx context.Context, cert *cmapi.Certificate, client cmclient.Interface) {
 	t.Helper()
-	testCond := cmapi.CertificateCondition{
-		Type:    cmapi.CertificateConditionType("Test"),
-		Reason:  "TestTwo",
```

Comment:
Looks like you changed the Reason "TestTwo" > "Test". Does that matter?

Reply:
No, I think I called it `TestTwo` when I originally wrote that test and had some issues whilst running it and for some reason already had a cert named `Test` in cluster that I didn't want to delete :)